engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/NadelEngineExecutionHooks.kt

```diff
@@ -11,4 +12,8 @@ interface NadelEngineExecutionHooks : ServiceExecutionHooks {
         parentNode: JsonNode,
         aliasHelper: NadelAliasHelper
     ): T?
+
+    fun <T> partitionArgumentList(args: List<T>, instruction: NadelBatchHydrationFieldInstruction): List<List<T>> {
```

Comment:
Can you make this function name more specific? From just reading it, you don't get any idea of what it does.

That or add some documentation.

Reply:
is it a dead function ?  its just listOf() a list right now

Is this missing code or dead idea code?

Reply:
Ahh ignore me - this is a default method - yeah so doco on what a hook would be expected to do is crucial

Also - is there no more context the hook may need?

Reply:
added more docs, don't think we need anything else, do we?